README.md

```diff
@@ -18,13 +18,19 @@ This VS Code extension from the Modular team adds support for the
 
 ## Get started
 
-1. [Install the MAX SDK.](https://docs.modular.com/max/get-started/)
-2. Open the folder you installed MAX in.
-3. Open any `.mojo` or `.🔥` file.
+1. [Install Mojo](https://docs.modular.com/mojo/manual/install).
+2. [Install the Mojo VS Code extension](https://marketplace.visualstudio.com/items?itemName=modular-mojotools.vscode-mojo).
+3. Open any `.mojo` or `.🔥` file and start coding.
 
-### Mojo SDK Resolution
+### Mojo SDK resolution
 
-The extension relies on the Python extension for locating your Python environment. In some cases, this appears to default to your globally-installed environment, even when a virtual environment exists. If the Mojo extension cannot find your SDK installation, try invoking the `Python: Set Project Environment` command and selecting your virtual environment.
+The extension relies on the Python extension for locating your Python
+environment. In some cases, this appears to default to your globally-installed
+environment, even when a virtual environment exists.
+
+If the Mojo extension cannot find your SDK installation, try invoking the
+`Python: Set Project Interpreter` command and selecting your virtual
```

Comment:
The exact command is `Python: Select Interpreter`. I'll close my PR changing this.

Reply:
Oops thank you!